src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -27,13 +30,15 @@ const ChatItemMyTextV2 = ({
   time = '4h',
   isContinuous = false,
   message = '',
+  attachments = [],
   status = ChatStatus.PENDING,
   avatar,
   chatType,
   messageType,
   data
 }: ChatItemMyTextProps) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `attachments = []` in the destructuring of props is necessary to handle the new feature of attachments in chat messages. The `onOpenMediaPreview` function from `useMessageHook` is also imported to handle opening media previews when an attachment is clicked.
```diff
+   attachments = [],
+   const {setReplyPreview, onContextMenuPressed, onOpenMediaPreview} = useMessageHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->